File: src/AppBundle/Entity/Emplacement.php
<?php
namespace AppBundle\Entity;

use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;

/**
 * Emplacement
 *
 * @ORM\Table(name="emplacement")
 * @ORM\Entity(repositoryClass="AppBundle\Repository\EmplacementRepository")
 */
class Emplacement
{
    /**
     * @var int $_iIdEmplacement
     *
     * @ORM\Column(name="id_emplacement", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $_iIdEmplacement;

    /**
     * @var string $_sNameEmplacement
     * @Assert\NotBlank()
     * @ORM\Column(name="name_emplacement", type="string", length=255)
     */
    private $_sNameEmplacement;

    /**
     * @var float $_fPriceEmplacement
     * @Assert\NotBlank()
     * @ORM\Column(name="price_emplacement", type="float")
     */
    private $_fPriceEmplacement;

    /**
     * @var integer $_iSurfaceEmplacement
     * @Assert\NotBlank()
     * @ORM\Column(name="surface_emplacement", type="integer")
     */
    private $_iSurfaceEmplacement;

     /**
     * @var integer $_iNbMaxPersonsEmplacement
     * @ORM\Column(name="nb_max_persons_emplacement", type="integer", nullable=true)
     */
    private $_iNbMaxPersonsEmplacement;
    
     /**
     * @var integer $_iNbRoomsEmplacement
     * @ORM\Column(name="nb_rooms_emplacement", type="integer", nullable=true)
     */
    private $_iNbRoomsEmplacement;

    /**
     * @ORM\Column(name="lat_emplacement", type="string", nullable=true)
     */
    private $_sLatEmplacement;

    /**
     * @ORM\Column(name="lng_emplacement", type="string", nullable=true)
     */
    private $_sLngEmplacement;

    

    /**
     * @return int
     */
    public function getIdEmplacement()
    {
        return $this->_iIdEmplacement;
    }

    /**
     * @param int $iIdEmplacement
     */
    public function setIdEmplacement($iIdEmplacement)
    {
        $this->_iIdEmplacement = $iIdEmplacement;
    }

    /**
     * @return string
     */
    public function getNameEmplacement()
    {
        return $this->_sNameEmplacement;
    }

    /**
     * @param string $sNameEmplacement
     */
    public function setNameEmplacement($sNameEmplacement)
    {
        $this->_sNameEmplacement = $sNameEmplacement;
    }

    /**
     * @return float
     */
    public function getPriceEmplacement()
    {
        return $this->_fPriceEmplacement;
    }

    /**
     * @param float $fPriceEmplacement
     */
    public function setPriceEmplacement($fPriceEmplacement)
    {
        $this->_fPriceEmplacement = $fPriceEmplacement;
    }

    /**
     * @return int
     */
    public function getSurfaceEmplacement()
    {
        return $this->_iSurfaceEmplacement;
    }

    /**
     * @param int $iSurfaceEmplacement
     */
    public function setSurfaceEmplacement($iSurfaceEmplacement)
    {
        $this->_iSurfaceEmplacement = $iSurfaceEmplacement;
    }

    /**
     * @return int
     */
    public function getNbMaxPersonsEmplacement()
    {
        return $this->_iNbMaxPersonsEmplacement;
    }

    /**
     * @param int $iNbMaxPersonsEmplacement
     */
    public function setNbMaxPersonsEmplacement($iNbMaxPersonsEmplacement)
    {
        $this->_iNbMaxPersonsEmplacement = $iNbMaxPersonsEmplacement;
    }

    /**
     * @return int
     */
    public function getNbRoomsEmplacement()
    {
        return $this->_iNbRoomsEmplacement;
    }

    /**
     * @param int $iNbRoomsEmplacement
     */
    public function setINbRoomsEmplacement($iNbRoomsEmplacement)
    {
        $this->_iNbRoomsEmplacement = $iNbRoomsEmplacement;
    }

    /**
     * @return string
     */
    public function getLatEmplacement()
    {
        return $this->_sLatEmplacement;
    }

    /**
     * @param string $sLatEmplacement
     */
    public function setLatEmplacement($sLatEmplacement)
    {
        $this->_sLatEmplacement = $sLatEmplacement;
    }

    /**
     * @return string
     */
    public function getLngEmplacement()
    {
        return $this->_sLngEmplacement;
    }

    /**
     * @param string $sLngEmplacement
     */
    public function setLngEmplacement($sLngEmplacement)
    {
        $this->_sLngEmplacement = $sLngEmplacement;
    }

}

File: app_mobiles/Camping_imerir/app/src/main/java/com/example/student/camping_imerir/models/Emplacement.java
package com.example.student.camping_imerir.models;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

/**
 * Created by student on 30/10/2017.
 */

public class Emplacement extends RealmObject {

    @PrimaryKey
    private int id_emplacement;
    private String name_emplacement;
    private String price_emplacement;
    private int surface_emplacement;
    private int nb_persons_emplacemenent;
    private int nb_rooms_emplacement;
    private String lat_emplacement;
    private String lng_emplacement;

    public Emplacement(){

    }

    public int getId_emplacement() {
        return id_emplacement;
    }

    public void setId_emplacement(int id_emplacement) {
        this.id_emplacement = id_emplacement;
    }

    public String getName_emplacement() {
        return name_emplacement;
    }

    public void setName_emplacement(String name_emplacement) {
        this.name_emplacement = name_emplacement;
    }

    public String getPrice_emplacement() {
        return price_emplacement;
    }

    public void setPrice_emplacement(String price_emplacement) {
        this.price_emplacement = price_emplacement;
    }

    public int getSurface_emplacement() {
        return surface_emplacement;
    }

    public void setSurface_emplacement(int surface_emplacement) {
        this.surface_emplacement = surface_emplacement;
    }

    public int getNb_persons_emplacemenent() {
        return nb_persons_emplacemenent;
    }

    public void setNb_persons_emplacemenent(int nb_persons_emplacemenent) {
        this.nb_persons_emplacemenent = nb_persons_emplacemenent;
    }

    public int getNb_rooms_emplacement() {
        return nb_rooms_emplacement;
    }

    public void setNb_rooms_emplacement(int nb_rooms_emplacement) {
        this.nb_rooms_emplacement = nb_rooms_emplacement;
    }

    public String getLat_emplacement() {
        return lat_emplacement;
    }

    public void setLat_emplacement(String lat_emplacement) {
        this.lat_emplacement = lat_emplacement;
    }

    public String getLng_emplacement() {
        return lng_emplacement;
    }

    public void setLng_emplacement(String lng_emplacement) {
        this.lng_emplacement = lng_emplacement;
    }
}
